infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -325,4 +324,22 @@ public void shutdown() {
         // Shut down the Corfu Runtime.
         corfuRuntime.cleanup(CorfuRuntime::shutdown);
     }
+
+    /**
+     * Management Server configuration
+     */
+    @Builder
+    @Getter
+    public static class ManagementServerConfig {
+        private final NodeLocator nodeLocator;
+
+        public static ManagementServerConfig parse(Map<String, Object> opts) {
+            final String address = (String) opts.get("--address");
+            final Integer port = (Integer) opts.get("<port>");
+
+            return ManagementServerConfig.builder()
+                    .nodeLocator(NodeLocator.parseString(address + ":" + port))
```

Comment:
Please check if this entire class actually used by ManagementServer, looks like localEndpoint variable not used by any one.

If it's not used then just delete this class and localEndpoint parameter.
It it's used by some one then you could use `NodeLocator.builder().host().port().build()` instead of parseString

Reply:
moved all to sever context.
done.